.gitattributes

```diff
@@ -0,0 +1,3 @@
+/.gitattributes export-ignore
```

Comment:
Waar wordt deze file voor gebruikt?

Reply:
`.gitattributes` is een Git file om dingen te configureren. Met `export-ignore` zorg je er voor dat GitHub in zijn zip-files deze bestanden weglaat. Dit wilde ik even uitproberen op onze bundles. Symfony doet dit ook en laat config files (php-cs-fixer.dist, phpunit.xml.dist, etc.), readme's, tests, etc. weg. Dat scheelt weer in de hoeveelheid files die keer op keer heen en weer worden gepompt